test/release/examples/primers/locales.chpl

```diff
@@ -214,18 +214,18 @@ class Data {
   var x:int;
 }
 
-var myData:unmanaged Data; // myData is a pointer-to-nil stored on Locale 0
+var myData: unmanaged Data; // myData is a pointer-to-nil stored on Locale 0
 
 on Locales[1 % numLocales] {
-  writeln("at start of on 1, myData is on locale ", myData.locale.id);
+  writeln("at start of 'on Locales[1]', myData is on locale ", myData.locale.id);
```

Comment:
Isn't it on Locales[0] when there is only one locale?  Maybe just say "the `on` statement"